src/axom/slam/docs/sphinx/index.rst

```diff
@@ -3,14 +3,16 @@
 .. ##
 .. ## SPDX-License-Identifier: (BSD-3-Clause)
 
+=======================
 Slam User Documentation
 =======================
 
 Axom's Set-theoretic Lightweight API for Meshes (SLAM) component provides high performance
 building blocks for distributed-memory mesh data structures in HPC simulation codes.
 
-Introduction
-------------
+.. raw:: html
```

Comment:
This is how I "fixed" the Slam user docs section headings.
(The nested `toc` calls in our unified docs promoted the heading in the index.rst file)

I'd be happy to hear if there is a cleaner way to do this.
If people are happy with this approach, I can apply this to the other affected components.